### PR TITLE
Update client error

### PR DESCRIPTION
**Pull request by morsedan**
I'm wondering if the 4 changes in the last Commit (17224e1b5b97f61df92a0ba52b036b60ed424d5d) would be better as ClientError.genericErrorString. Objectable is a protocol that can be enforced on different types, so there's no way to know what was being queried for. I went with "Failed to retrieve this object" and "Failed to retrieve these objects". I think "We're having some difficulty, please try again later" is friendlier and just as applicable.

### CODE REVIEW

File: Pods/Manifest.lock [hunk 1]
```diff
@@ -76,6 +76,6 @@ SPEC CHECKSUMS:
   TwilioAccessManager: f3b0d9edd2eb5a044e50ff78057ab13611a0baa1
   TwilioChatClient: 23168c2df8e306da1cedcad1d60ba69862235358
 
-PODFILE CHECKSUM: 098698bf79ea995754771d8f943f0bae44c3db01
+PODFILE CHECKSUM: 6470dd4b953323882f360bba41d437d82cbb3d62
```
Comment (Benjamindodgson) [inline Pods/Manifest.lock]: What changed in the pod file?

File: Pods/Target Support Files/Pods-Benji/Pods-Benji-frameworks-Debug-input-files.xcfilelist [hunk 2]
```diff
@@ -10,7 +10,5 @@ ${BUILT_PRODUCTS_DIR}/TMROExtensionObject/TMROExtensionObject.framework
 ${BUILT_PRODUCTS_DIR}/TMROFutures/TMROFutures.framework
 ${BUILT_PRODUCTS_DIR}/TMROLocalization/TMROLocalization.framework
 ${PODS_ROOT}/TwilioAccessManager/TwilioAccessManager.framework
-${PODS_ROOT}/TwilioChatClient/TwilioChatClient.framework
```
Comment (Benjamindodgson) [inline Pods/Target Support Files/Pods-Benji/Pods-Benji-frameworks-Debug-input-files.xcfilelist]: Hmmm. Not sure what caused this. Have you run the project?

File: Source/API/Connection+Cloud.swift [hunk 3]
```diff
@@ -27,7 +27,7 @@ struct CreateConnection: CloudFunction {
                                 } else if let connection = object as? Connection {
                                     promise.resolve(with: connection)
                                 } else {
-                                    promise.reject(with: ClientError.generic)
+                                    promise.reject(with: ClientError.message(detail: "There was a problem connecting with that phone number"))
```
Comment (Benjamindodgson) [inline Source/API/Connection+Cloud.swift]: Let's make sure all of these have periods.

Comment (morsedan) [reply Source/API/Connection+Cloud.swift]: done

File: Pods/Manifest.lock (hunk 1 above)
Comment (morsedan) [reply Pods/Manifest.lock]: Yes, this is weird. I think it is related to the comment below. Uninstalling and reinstalling TwilioChatClient resolved the issue.

File: Pods/Target Support Files/Pods-Benji/Pods-Benji-frameworks-Debug-input-files.xcfilelist (hunk 2 above)
Comment (morsedan) [reply Pods/Target Support Files/Pods-Benji/Pods-Benji-frameworks-Debug-input-files.xcfilelist]: The project builds and runs now. I'm not sure what happened here. I'll close this and create a new PR with the requested changes.

**Comment by morsedan** [comment]
Closing this request and opening a new one with the requested changes.

**Pull request closed without merging**